lib/generators/curation_concerns/templates/db/seeds.rb

```diff
@@ -0,0 +1,94 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+user = User.create(email: 'foo@example.org', password: 'foobarbaz')
+
+# Public
+3.times do |i|
+  GenericWork.create(title: ["Public #{i}"], read_groups: ['public']) do |work|
```

Comment:
We need real objects here. This is only used if you want/need fixture data in your environment; it's not run by default.